### ISSUE TITLE
Missing files after installing

### ISSUE THREAD

**Issue by romaincointepas** [Squirrel/Squirrel.Windows#768]
I'm encountering a really weird bug using [electron/grunt-electron-installer](https://github.com/electron/grunt-electron-installer): some directories' content is not being copied/installed in `app` after running the installer.

I'm running [electron-userland/electron-packager](https://github.com/electron-userland/electron-packager) followed by [electron/grunt-electron-installer](https://github.com/electron/grunt-electron-installer) on a Windows 10 32-bit VM.

Everything works fine except that the following directories' content is not being installed/copied on install:

`my-app/app-0.0.1/resources/app/node_modules/bittorrent-tracker/lib/client`
`my-app/app-0.0.1/resources/app/node_modules/bittorrent-tracker/lib/server`

I've also tried to copy/paste in different paths before running `grunt-electron-installer` to see if it was a path thing, all of the above are also empty.
`my-app/app-0.0.1/resources/app/bittorrent-tracker/lib/client`
`my-app/app-0.0.1/resources/app/bittorrent-tracker/lib/server`
`my-app/app-0.0.1/resources/app/lib/client`
`my-app/app-0.0.1/resources/app/lib/server`

However, those ones *work*:
`my-app/app-0.0.1/resources/app/client`
`my-app/app-0.0.1/resources/app/server`

And those ones work as well (renaming `lib` to `timmy`):
`my-app/app-0.0.1/resources/app/bittorrent-tracker/timmy/client`
`my-app/app-0.0.1/resources/app/bittorrent-tracker/timmy/server`

But those ones don't work (renaming `client` or `server` to `timmy`):
`my-app/app-0.0.1/resources/app/bittorrent-tracker/lib/timmy`

I've also tried adding dummy files/folders inside those directories, they are also not being copied (so it probably has to do with the path, not the actual files).

The installer also appears to create a weird `lib` folder in the `app-0.0.1` folder (this `lib` folder is not in the original build directory).

To clarify, here are the steps:
- Running `electron-packager`: the files are there in the resulting build folder.
- Running `grunt-electron-installer`
- Installing from resulting Setup.exe: the directories mentioned above are empty (in the `AppData/Local/...` path)

Here are version info:
```
+-- grunt-electron-installer@2.1.0
| `-- electron-winstaller@2.3.2
|   +-- bluebird@3.4.1
|   +-- fs-extra@0.26.7
|   `-- lodash.template@4.2.5
|     +-- lodash._reinterpolate@3.0.0
|     +-- lodash.assigninwith@4.0.7
|     | `-- lodash.keysin@4.1.4
|     +-- lodash.keys@4.0.7
|     +-- lodash.rest@4.0.3
|     +-- lodash.templatesettings@4.0.1
|     | `-- lodash.escape@4.0.0
|     `-- lodash.tostring@4.1.3
```

And grunt task:
```coffeescript
'create-windows-installer':
      win32_ia32:
        appDirectory: 'dist/builds/MyApp-win32-ia32'
        outputDirectory: 'dist/releases/win'
        exe: 'MyApp.exe'
        setupExe: 'MyAppInstaller.exe'
        noMsi: true
```

Note:
- I'm not using `asar` when building with `electron-packager`
- I've tried on 2 different VMs, same thing.
- It's not a depth bug (paths with higher depth work, and also the above tests suggest that it's not)

**My guess is that something funky is happening with folder named `lib`.**

**Comment by romaincointepas** [comment]
This seems to be related to https://github.com/Squirrel/Squirrel.Windows/issues/765.

**Comment by ruZZil** [comment]
Look into npm shrinkwrap to solve this issue for now. 
Run npm shrinkwrap and modify the electron-winstaller entry to use 2.3.1
Once electron-winstaller pulls the fix, you could use ^2.3.3
